src/SSW.SophieBot/SSWSophieBot/SSWSophieBot.dialog

```diff
@@ -812,11 +812,11 @@
             },
             {
               "property": "$options.location",
-              "value": "=@location"
+              "value": "=titleCase(@location)"
             },
             {
               "property": "$options.skill",
-              "value": "=@skill"
+              "value": "=titleCase(@skill)"
```

Comment:
We capitalized them first then discuss with the team, because most of the skill names need capitalization, it's better than not do it at all. Do you agree?